windows/src/global/delphi/keyboards/Keyman.System.KeyboardJSInfo.pas

```diff
@@ -0,0 +1,104 @@
+unit Keyman.System.KeyboardJSInfo;
+
+interface
+
+uses
+  System.Classes,
+  System.SysUtils;
+
+type
+  TKeyboardJSInfo = class
+  private
+    FVersion: string;
+    FName: string;
+    FMnemonic: Boolean;
+    FRTL: Boolean;
+
```

Comment:
What about parsing for `this.KI` and including `keyboardID` in this struct?

Reply:
`keyboardID` is already captured from the name of the file. We can do this if we need it at a future date.